cookiecutter-django-ida/{{cookiecutter.repo_name}}/requirements/constraint.txt

```diff
@@ -1,12 +0,0 @@
-# Version constraints for pip-installation.
-#
-# This file doesn't install any packages. It specifies version constraints
-# that will be applied if a package is needed.
-#
-# When pinning something here, please provide an explanation of why.  Ideally,
-# link to other information that will help people in the future to remove the
-# pin when possible.  Writing an issue against the offending project and
-# linking to it here is good.
-
-# code-annotations currently restricts to Django<2.3
-Django<2.3
```

Comment:
You shouldn't remove the file, just this line since the constraints file is referenced by other files.

Reply:
oh, wait I see what you mean now

Reply:
Look again: https://github.com/edx/edx-cookiecutters/blob/timmc/constraint/cookiecutter-django-ida/%7B%7Bcookiecutter.repo_name%7D%7D/requirements/constraints.txt (constraint vs. constraints -- the file is doubled up)